packages/cli/src/commands/deploy.ts

```diff
@@ -261,3 +281,43 @@ const isBuildable = (path: string) => {
 
   return true
 }
+
+const selectApiScope = async (token: string): Promise<Scope> => {
+  const accessibleToMySpace = await StoryblokClient({
```

Comment:
There are some variables that are named "MySpace". They could be refactored.

Reply:
Nice catch!